recipes/openmeeg/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+cmake $SRC_DIR -DBLA_VENDOR=OpenBLAS -DENABLE_PYTHON=ON -DCMAKE_BUILD_TYPE=RELEASE -DBUILD_DOCUMENTATION=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_CXX_COMPILER=$PREFIX/bin/clang++ -DCMAKE_C_COMPILER=$PREFIX/bin/clang
```

Comment:
Would wrap this to make it easier to read like [so]( https://github.com/conda-forge/gflags-feedstock/blob/master/recipe/build.sh ).